### DISCUSSION TITLE
Terms of Use

### DISCUSSION THREAD

**Discussion by AnastasiaE95** [community/community#65607] [New to GitHub] [answered]
Hello! I would like to enquire about the website's terms of use:

1) Could I create my own content on the website and post hyperlinks to it on commercial platforms?
2) Could I also create derivative works? I am particularly interested in whether I could:
A. create screencasts and video reviews of my content on the website and use them commercially;
B. take screenshots (for example, the homepage or my own account on the website) and use them commercially.

Looking forward to hearing from you.

**Comment by queenofcorgis** [comment, marked as answer]
Apologies for the delay @AnastasiaE95 - I was checking with some folks to make sure you had the proper information.

>Could I create my own content on the website and post hyperlinks to it on commercial platforms? 

Yes, you are allowed to do so. 

> Could I also create derivative works? I

We do not allow the use of others to use our GitHub assets for commercial purposes (ie the homepage) so this would not be possible.

**Reply by AnastasiaE95** [reply]
@queenofcorgis Hello, Lili! Thank you for the answer. 

Is there a way to acquire permission to use your GitHub assets for commercial purposes, e. g. by signing a contract and/or agreement? I work for a commercial online school, and we are currently working on a course for teenage developers to help them learn about various platforms and tools they could use at work in the future. 

Looking forward to hearing from you.

**Reply by queenofcorgis** [reply]
Hi there - the best way to send a request to acquire permission is to contact GitHub through their [support page](https://support.github.com/?tags=dotcom-direct). However, please keep in mind that this is not a guarantee that your request will be approved.

**Reply by AnastasiaE95** [reply]
@queenofcorgis Lili, thank you for the link. I have seen this page before; however, it seems that there is no category suitable for my request :( This link https://support.github.com/request?tags=docs-policy leads me to the homepage as well.

**Reply by queenofcorgis** [reply]
What about this one: https://support.github.com/contact?tags=dotcom-direct%2Crr-general-technical

**Reply by AnastasiaE95** [reply]
@queenofcorgis Lili, this one is working. Thanks for your help!

**Reply by queenofcorgis** [reply]
You're welcome - thanks for your patience with the links 😃

**Reply by AnastasiaE95** [reply]
Hopefully, I will hear back from the support team soon. Have a nice day :)

**Reply by queenofcorgis** [reply]
Thanks - you too! And thanks for marking the answer - hope we continue to see you around the community 💃🏼
